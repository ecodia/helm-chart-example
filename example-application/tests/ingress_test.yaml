suite: test ingress
templates:
  - ingress.yaml
tests:
  - it: should render nothing if not disabled
    set:
      ingress:
        enabled: false
    asserts:
      - hasDocuments:
          count: 0
  - it: should be of type ingress
    set:
       ingress:
         enabled: true
    asserts:
      - isKind:
          of: Ingress
  - it: manifest should match snapshot
    set:
      ingress:
        enabled: true
      image:
        repository: test-docker-registry
        tag: 1.2.3-TEST
    asserts:
      - matchSnapshot:
          path: spec
