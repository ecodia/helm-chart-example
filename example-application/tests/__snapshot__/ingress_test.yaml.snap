manifest should match snapshot:
  1: |
    rules:
    - host: chart-example.local
      http:
        paths: null
